backend/satellite_tools/reposync.py

```diff
@@ -684,9 +684,9 @@ def sync(self, update_repodata=True):
         fileutils.createPath(os.path.join(CFG.MOUNT_POINT, 'rhn'))  # if the directory exists update ownership only
         for root, dirs, files in os.walk(os.path.join(CFG.MOUNT_POINT, 'rhn')):
             for d in dirs:
-                fileutils.setPermsPath(os.path.join(root, d), group=APACHE_GROUP)
+                fileutils.setPermsPath(os.path.join(root, d), group=APACHE_GROUP, chmod=int('0755', 8))
             for f in files:
-                fileutils.setPermsPath(os.path.join(root, f), group=APACHE_GROUP)
+                fileutils.setPermsPath(os.path.join(root, f), group=APACHE_GROUP, chmod=int('0754', 8))
```

Comment:
`0754` for "files" is very unusual. Or did you want to set it to `0644` ?

Reply:
I have actually just taken the existing permissions and added the `read` rights for `other`. `644` will also work for the purpose of this PR. I am unsure of any other impact of removing `execute` rights for `owner`/`group`.

Reply:
I think we should try with `0644` for the files. For file transfer we should not need execute permissions.